Description: Added missed includes
  just copied the respective upstream lines
Author: Alf Gaida <agaida@siduction.org>

Last-Update: 2020-02-08

--- nomacs-3.12.0+dfsg.orig/src/DkCore/DkMetaData.cpp
+++ nomacs-3.12.0+dfsg/src/DkCore/DkMetaData.cpp
@@ -42,6 +42,8 @@
 #include <QApplication>
 #pragma warning(pop)		// no warnings from includes - end
 
+#include <iostream>
+
 namespace nmc {
 
 // DkMetaDataT --------------------------------------------------------------------
--- nomacs-3.12.0+dfsg.orig/src/DkGui/DkNoMacs.cpp
+++ nomacs-3.12.0+dfsg/src/DkGui/DkNoMacs.cpp
@@ -96,6 +96,8 @@
 #include <QWinTaskbarButton>
 #endif
 
+#include <iostream>
+
 namespace nmc {
 
 DkNomacsOSXEventFilter::DkNomacsOSXEventFilter(QObject *parent) : QObject(parent) {
